client/my-sites/hosting/github/index.tsx

```diff
@@ -18,7 +18,7 @@ export const GitHubCard = () => {
 	const gitHub = connections.find( ( connection ) => connection.service === 'github-deploy' );
 
 	if ( gitHub ) {
-		return <GithubConnectCard connection={ gitHub } />;
+		return <DeploymentCard repo="Automattic/testing" branch="main" repoUrl="" />;
```

Comment:
I think we have the necessary information exposed already. If we don't let's keep it like this is right now and address it in a follow-up PR. Let's remove the `repoUrl` argument 👍